### PR TITLE
fix(engines): support node 10+ and update prepublish script

### LINKED ISSUE #223: Node version incompatibility
Is it possible to support node versions greater than 12.x.x?

### PR DESCRIPTION
**Pull request by ruisaraiva19**
fix #223

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -72,7 +72,7 @@
     "react": ">=0.14.1"
   },
   "engines": {
-    "node": "^10.13.0 || ^12.x",
+    "node": "^10.13.0 || >= 12.x",
```
Comment (tjallingt) [inline package.json:75]: what would the difference be with:  `>= 10.x` or event being very explicit `10.x || 11.x || 12.x || 13.x`.

Comment (ruisaraiva19) [reply package.json:75]: With `>= 10.x` will match Node.js version 10 and up. With `10.x || 11.x || 12.x || 13.x` does the same as `>= 10.x` but will not match upcoming versions (Node.js 14 is around the corner).

**Comment by ruisaraiva19** [comment]
@tjallingt as a reminder, we need to merge `master` into `canary` to fix the tags on the `canary` branch before we merge this PR.

File: package.json (hunk 1 above)
Comment (ruisaraiva19) [reply package.json:75]: I can update it to `>= 10.x` and the tests pipeline accordingly.

**Comment by kailan** [comment]
Hi there, do you have a rough timeframe on merging this? My build is broken at the minute. Thanks

File: package.json (hunk 1 above)
Comment (ruisaraiva19) [reply package.json]: I pushed a commit with the `node` engine property updated to `>= 10.x`.

**Comment by tjallingt** [comment]
Yo sorry for not checking on this! Didn't realize it had this priority. Cant fix up canary right now, will do asap

**Comment by kailan** [comment]
Thanks very much guys! Much appreciated.

**Pull request merged**